exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -229,9 +229,15 @@ def __init__(
 
         self._headers = headers or environ.get(OTEL_EXPORTER_OTLP_HEADERS)
         if isinstance(self._headers, str):
-            self._headers = tuple(
-                tuple(item.split("=")) for item in self._headers.split(",")
-            )
```

Comment:
Your change is totally fine but you could just add another list comprehension to the existing one. Something like

```
[(key.lower(), value) for key, value in [item.split("=") for item in headers.split(",")]]
```

since this runs only once, the extra pass over the collection should not be problem. That said, a plain loop may be more obvious/readable to some over nested comprehension. I'll leave it to you.

Reply:
I appreciate the feedback! I also thought about just adding another nested list comp, but it was getting a bit out of hand, so I made it into a plain loop! Thanks again 😄